crates/api/src/v1/package.rs

```diff
@@ -112,11 +112,8 @@ pub enum PackageError {
     #[error("the record is not currently sourcing content")]
     RecordNotSourcing,
     /// The publish operation was not authorized by the registry.
-    #[error("registry did not authorize the request to publish: {message}")]
-    Unauthorized {
-        /// The error message.
-        message: String,
```

Comment:
I can standardize the error types in the other direction instead (there were other tuple variants and this one was the outlier) and make them all struct variants.

Reply:
I guess if it's specifically a convention that "error enums with a single message field are mono-tuples" that makes sense to me. I just don't want to lose names when they can't be inferred so easily.

Speaking of which, it might make sense to document some conventions at some point.